storage/acl.go

```diff
@@ -314,6 +345,15 @@ func (r ACLRule) toRawObjectAccessControl(bucket string) *raw.ObjectAccessContro
 	}
 }
 
+func (r ACLRule) toProtoObjectAccessControl(bucket string) *storagepb.ObjectAccessControl {
+	return &storagepb.ObjectAccessControl{
+		Entity: string(r.Entity),
+		Role:   string(r.Role),
+		// The other fields are not settable.
+		// TODO: Is that the case for gRPC?
```

Comment:
Yes, only these fields can be updated in JSON https://cloud.google.com/storage/docs/json_api/v1/objectAccessControls/patch

...and really the other fields only make sense as read-only.

Reply:
Ok so I will remove the TODO.